### PR TITLE
Add a howto on issuing manual multi-sig dispute payouts

### PR DESCRIPTION
**Pull request by cbeams**
We had an actual arbitration case come up requiring a manual payout using Bisq's "Emergency multi-sig payout tool". The tool has been around for a while, and used several times, but never documented. This "howto" doc is the first of its kind in the bisq-network/docs repository, and is intended to set a precendent for future such docs that walk through specific use cases with the Bisq exchange, or in the future, the Bisq DAO.

@ManfredKarrer, please review the text for accuracy. I adapted this from the Google Doc you put together (thanks for that!), but it'll be good to have you double-check that I got everything right.

I'll be following these steps myself as I work with the trader in question who needs this payout.

So **please do not merge this PR** until I've completed the manual payout and given the go ahead. That way I will be able to correct any issues along the way.

PS: in case it's not obvious, the rendered version of this document can be viewed in my fork at https://github.com/cbeams/bisq-docs/blob/manual-payout/exchange/howto/manual-dispute-payout.adoc.

### CODE REVIEW

File: exchange/howto/manual-dispute-payout.adoc [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+depositTxHex:: The Deposit transaction ID from the Dispute details view, i.e. the same transaction ID you clicked on to view the transaction in a block explorer.
+
+buyerPayoutAmount:: The amount in BTC that the buyer should be paid out, as noted above, e.g. `0.03`
+
+sellerPayoutAmount:: The amount in BTC the seller should be paid out, e.g. `0.0662`
+
+arbitratorPayoutAmount:: This value should always be `0`, as we no longer issue payouts to arbitrators
+
+Tx fee:: The BTC value of the `txFee` in Satoshis as taken from the JSON contract details above, e.g. `0.00036`
```
Comment (ManfredKarrer) [inline exchange/howto/manual-dispute-payout.adoc]: Suggest: 
The BTC value of the `txFee` in Bitcoin as taken from the JSON contract details above (convert Satoshi value to Bitcoin), e.g. `0.00036`

File: exchange/howto/manual-dispute-payout.adoc [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+When all values have been entered (and double-checked!), click `Sign and publish transaction` to issue the payout.
+
+When you have confirmed that the payout transaction worked as expected, *let the trader know and close out the ticket*.
+
+
+== Trader instructions
+
+[WARNING]
+The instructions below require you to share sensitive private key information with your arbitrator in order to assist in a manual multi-sig payout. This is necessary only in extraordinary situations, e.g. when a bug or network issue has caused the normal, automated multi-sig payout process to fail. In the unlikely event that you do need to follow these instructions, *you should never share private key information with anyone other than your arbitrator*, and *you should send your private key information only via the Bisq support ticket chat interface*.
```
Comment (ManfredKarrer) [inline exchange/howto/manual-dispute-payout.adoc]: Suggested addition:
The private key the arbitrator requests from you is only used for payout from the  MultiSig address of that particular trade. All other trades have different addresses and keys. For Bitcoins you are holding in your Bisq wallet other addresses and keys are used as well.

File: exchange/howto/manual-dispute-payout.adoc [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+At the bottom of the dialog, *check the box that reads `Include private keys`*:
+
+image:images/include-private-keys.png[include private keys]
+
+Then *click `Copy to clipboard`*.
+
+Open the text editor of your choice, e.g. TextEdit on macOS or Notepad on Windows, and *paste the copied wallet data text* into it.
+
+In the text editor, *search for the public key value your arbitrator provided* in step one above.
```
Comment (ManfredKarrer) [inline exchange/howto/manual-dispute-payout.adoc]: Be sure the found text gets followed by "DeterministicKey{pub HEX=....". There are also all transactions listed in the text and you might find the public key there as well. Continue your search until you are at the address entry with the keys.

File: exchange/howto/manual-dispute-payout.adoc [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+
+When you have found it, select and *copy the _next_ line in the file*—the one that that reads `priv HEX=...`. *This value is your private key*.
```
Comment (ManfredKarrer) [inline exchange/howto/manual-dispute-payout.adoc]: ...one that reads....

**Comment by cbeams** [comment]
@ManfredKarrer, your feedback has been addressed in my most recent commits above, and for the record, these instructions were used successfully today to issue a manual payout.

Note especially the changes in e837a1f—I ran into an exception when attempting the payout the first time, and realized after debugging it that it was because both buyer and seller pubkeys are necessary, regardless of which one is actually participating in the payout transaction.

In any case, I am merging this now, as there are no new substantive changes beyond what you've already seen and what I just mentioned above.

**Comment by ManfredKarrer** [comment]
Ah yes, sorry forgot that you need all 3 Pubkeys.

**Pull request merged**